QUEST_LV_0100.tsv

```diff
@@ -7557,16 +7557,16 @@ QUEST_LV_0100_20150714_007556	Set the strings
 QUEST_LV_0100_20150714_007557	To prevent Pantos from going to other regions, please set the strings on Srautas Canyon and Nepritas Cliffs.
```

Comment:
"To prevent the Pantos from going to other regions, please set the strings on Srautas Canyon and Nepritas Cliffs."